backend/app/main.py

```
@@ -61,9 +61,71 @@ async def lifespan(app: FastAPI):
+        try:
+            db = SessionLocal()
+            try:
+                tokens_found = schwab_tokens_exist(db)
+            finally:
+                db.close()
+            if tokens_found:
+                raise EncryptionKeyMissing(
+                    "SCHWAB_ENCRYPTION_KEY env var is required when Schwab "
+                    "tokens exist in the database. Set this env var or remove "
+                    "existing tokens to start the application."
+                )
```

Ruff (0.15.4)

[warning] Abstract `raise` to an inner function

(TRY301)